force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -226,20 +233,18 @@ public with sharing class ClassesObjectsMethods {
      * where you need to manipulate and manage book information.
      */
     public class Book {
-        // Declare three public instance variables - title, author, pages
-        // title
-        // author
-        // pages
+        public String title;
+        public String author;
+        public Integer pages;
     
-        // Book constructor to initialize the title and author
-        public Book() {
-            // Initialize the title and author
+        public Book(String title, String author) {
+            this.title = title;
+            this.author = author;
         }
     
-        // Method that returns the details of the book
         public String getBookDetails() {
-            // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+            String str = 'Title: ' + title + ',' + ' ' + 'Author: ' + author;
```

Comment:
While not required in Apex, it's a good idea to use the `this` keyword when referencing instance variables in your code:
```suggestion
            String str = 'Title: ' + .this.title + ',' + ' ' + 'Author: ' + this.author;
```